README.md

```diff
@@ -45,7 +45,7 @@ The **Discovery Server tool documentation** can be found
 ## Installation Guide
 
 In order to use the Discovery Server tool, it is necessary to have a compatible version of
-[eProsima Fast DDS](https://eprosima-fast-rtps.readthedocs.io/en/latest/) installed (over release 2.0.2).
+[eProsima Fast DDS](https://fast-dds.docs.eprosima.com/en/latest/) installed (over release 2.0.2).
```

Comment:
```suggestion
[eProsima Fast DDS](https://fast-dds.docs.eprosima.com/en/latest/) installed (over release 3.0.0).
```

Or else have a compatibility versions table, though I do not think there are releases done in this repo and I doubt we will start doing them now (as it is a testing repository).